doc/source/index.rst

```diff
@@ -53,8 +59,8 @@ Ansys Electronics Desktop (AEDT) to make scripting simpler.
             AEDT API reference
             ^^^^^^^^^^^^^^^^^^
 
-            Understand PyAEDT API endpoints, their capabilities,
-            and how to interact with them programmatically.
+            The PyAEDT API reference contains a detailed description of the functions and modules included in PyAEDT.
+            It describes how the methods work and which parameter can be used.
```

Comment:
```suggestion
            The PyAEDT API reference contains descriptions of the functions and modules included in PyAEDT.
            It describes how the methods work and the parameter that can be used.
```
Avoid using "detailed" or "brief" and just describe what is present. In most cases, these adjectives aren't all that accurate!